File: java/Fibonacci.java
public class Fibonacci {
    static int n1=0,n2=1,n3=0;
    static void printFibonacci(int count){
        if(count>0){
            n3 = n1 + n2;
            n1 = n2;
            n2 = n3;

            System.out.println(n3);

            printFibonacci(count-1);
        }
    }
    public static void main(String args[]){
        printFibonacci(10);
    }

}

File: js/fibonacci.js
// Simple Algorithm to find the nth number of the fibonacci sequence.
function fib(nth) {
    var firstN = 0;
    var secondN = 1;
    
    if(nth < 1) {
    	throw "invalid nth num";
    }
    if(nth == 1) {
      return 0;
    } else if (nth == 2) {
      return 1;
    } else {
      for(var i = nth-2; i > 0; i--) {
        var tempN = firstN + secondN;
        firstN = secondN;
        secondN = tempN;
      }
      return secondN;
    }
    
}
